drivers/sensor/lis2dh/lis2dh_trigger.c

```diff
@@ -203,6 +258,8 @@ int lis2dh_trigger_set(const struct device *dev,
 		return lis2dh_trigger_drdy_set(dev, trig->chan, handler, trig);
 	} else if (trig->type == SENSOR_TRIG_DELTA) {
 		return lis2dh_trigger_anym_set(dev, handler, trig);
+	} else if (trig->type == SENSOR_TRIG_TAP) {
+		return lis2dh_trigger_anym_set(dev, handler, trig);
```

Comment:
Use a separate set routine here. Alternatively, use a single one, but with a different name (anym_set seems only for "any motion"). I would be more for a separation if not too difficult.